### PR TITLE
Compiling for Linux/*BSD: Add NetBSD one-liner

### LINKED ISSUE #4029: NetBSD needs to be added to the documentation
**Your Godot version:**
3.2.2 and 3.2.3

**Issue description:**
There needs to be instructions on compiling for NetBSD. A bug is being looked at for being able to compile it for NetBSD https://github.com/godotengine/godot/issues/42145 and adding NetBSD to the official documentation would be a big help.

**URL to the documentation page (if already existing):**
https://docs.godotengine.org/en/stable/development/compiling/compiling_for_x11.html

### PR DESCRIPTION
**Pull request by akien-mga**
Fixes #4029.

### CODE REVIEW

File: development/compiling/compiling_for_linuxbsd.rst [hunk 1]
```diff
@@ -79,6 +79,12 @@ Distro-specific one-liners
 |                  |     sudo zypper install scons pkgconfig libX11-devel libXcursor-devel libXrandr-devel libXinerama-devel \ |
 |                  |             libXi-devel Mesa-libGL-devel alsa-devel libpulse-devel libudev-devel libGLU1 yasm             |
 +------------------+-----------------------------------------------------------------------------------------------------------+
+| **NetBSD**       | ::                                                                                                        |
+|                  |                                                                                                           |
+|                  |     pkg_add pkg-config py37-scons yasm                                                                    |
+|                  |                                                                                                           |
+|                  | For audio support, you can optionally install ``alsa-lib`` and/or ``pulseaudio``.                         |
```
Comment (akien-mga) [inline development/compiling/compiling_for_linuxbsd.rst:86]: Note: I'm still building with those installed to confirm if they actually work.

Comment (akien-mga) [reply development/compiling/compiling_for_linuxbsd.rst:86]: PulseAudio seems to work fine. ALSA doesn't work out of the box, so I'll remove the recommendation.

**Comment by notepade** [comment]
Would Xcursor, Xinerama, Xi and XRandR need to be added to the install command? Because they don't come with X11 out the box. And don't forget python 3. Oh, and GLEW along with the other OpenGL libraries too.

**Comment by akien-mga** [comment]
> Would Xcursor, Xinerama, Xi and XRandR need to be added to the install command? Because they don't come with X11 out the box. And don't forget python 3. Oh, and GLEW along with the other OpenGL libraries too.

I didn't have to install anything else, so it seems they all came out of the box with the X11 flavor of NetBSD 9.0 for me.
Python 3 should be pulled in by `py37-scons` I assume. GLEW is not used since Godot 2.1.

**Comment by notepade** [comment]
> Python 3 should be pulled in by `py37-scons` I assume. GLEW is not used since Godot 2.1.
What about the MesaGL libraries? Does that not come with NetBSD by default?

**Comment by akien-mga** [comment]
> What about the MesaGL libraries? Does that not come with NetBSD by default?

I don't know, but I didn't have to install anything more than what I listed to compile and run Godot.

**Comment by notepade** [comment]
> > What about the MesaGL libraries? Does that not come with NetBSD by default?
> 
> I don't know, but I didn't have to install anything more than what I listed to compile and run Godot.

You didn't happen to run into "#error the "loader_platform.h" file must be modified for this OS"" error when compiling on NetBSD did you?

**Comment by akien-mga** [comment]
> You didn't happen to run into "#error the "loader_platform.h" file must be modified for this OS"" error when compiling on NetBSD did you?

Yes, with the `master` branch, see https://github.com/godotengine/godot/pull/42157#issuecomment-694788846

**Pull request merged**